graphql-kotlin-schema-generator/src/main/kotlin/com/expediagroup/graphql/generator/ClassScanner.kt

```diff
@@ -22,7 +22,7 @@ import io.github.classgraph.ClassInfoList
 import kotlin.reflect.KClass
 import kotlin.reflect.jvm.jvmName
 
-internal class SubTypeMapper(supportedPackages: List<String>) {
+internal class ClassScanner(supportedPackages: List<String>) {
```

Comment:
if you are renaming the class you should rename all of the props below to reflect it as well to avoid the confusion

Reply:
Done